k8s/k8s.go

```diff
@@ -26,6 +27,7 @@ type Client struct {
 	podController       *cache.Controller
 	podIndexer          cache.Indexer
 	nodeName            string
+	dealWithDupIP       bool
```

Comment:
```suggestion
	resolveDupIPs       bool
```

Consistency with above suggestions.